error: product 'SPMUtility' not found

I get the following error when trying to install with Homebrew

```
swift build --disable-sandbox -c release -Xswiftc "-target" -Xswiftc "x86_64-apple-macosx10.12"
Fetching https://github.com/apple/swift-package-manager.git
Fetching https://github.com/apple/swift-llbuild.git
Fetching https://github.com/apple/swift-tools-support-core.git
Cloning https://github.com/apple/swift-package-manager.git
Resolving https://github.com/apple/swift-package-manager.git at 0.6.0
Cloning https://github.com/apple/swift-tools-support-core.git
Resolving https://github.com/apple/swift-tools-support-core.git at 0.1.0
Cloning https://github.com/apple/swift-llbuild.git
Resolving https://github.com/apple/swift-llbuild.git at 0.3.0
'llbuild' /private/tmp/cbeacon-20201119-25125-eaaac9/cbeacon-0.4.1/.build/checkouts/swift-llbuild: warning: ignoring broken symlink /private/tmp/cbeacon-20201119-25125-eaaac9/cbeacon-0.4.1/.build/checkouts/swift-llbuild/lib/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm
'cbeacon' /private/tmp/cbeacon-20201119-25125-eaaac9/cbeacon-0.4.1: error: product 'SPMUtility' not found. It is required by target 'cbeacon'.
warning: dependency 'SwiftPM' is not used by any target
make: *** [build] Error 1
```

--- a/Package.swift
+++ b/Package.swift
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
     name: "cbeacon",
     dependencies: [
-        .package(url: "https://github.com/apple/swift-package-manager.git", from: "0.3.0")
+        .package(url: "https://github.com/apple/swift-package-manager.git", .exact( "0.4.0")),
     ],
     targets: [
         .target(

